### PR TITLE
Copter: Added new settings for flight mode channel

### PR DESCRIPTION
**Pull request by muramura**
The channels that can be configured as flight mode channels are 5-15.
I will add channels 9-15.

### CODE REVIEW

File: ArduCopter/Parameters.cpp [hunk 1]
```diff
@@ -311,7 +311,7 @@ const AP_Param::Info Copter::var_info[] = {
     // @Param: FLTMODE_CH
     // @DisplayName: Flightmode channel
     // @Description: RC Channel to use for flight mode control
-    // @Values: 0:Disabled,5:Channel5,6:Channel6,7:Channel7,8:Channel8
+    // @Values: 0:Disabled,5:Channel5,6:Channel6,7:Channel7,8:Channel8,9:Channel9,10:Channel10,11:Channel11,12:Channel12,13:Channel13,14:Channel14,15:Channel15
```
Comment (IamPete1) [inline ArduCopter/Parameters.cpp:314]: This would really benefit from some spaces, `11:Channel11` -> `11: Channel 11`

**Comment by muramura** [comment]
@IamPete1 san. I agreed. I've changed.

**Review by khancyr — APPROVED** [review]

File: ArduCopter/Parameters.cpp (hunk 1 above)
Comment (rmackay9) [reply ArduCopter/Parameters.cpp]: I'm being a bit nitpicky but I would make it "11:Channel 11".  I.e. no space between the "11:" and "Channel".

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**